tycho_demo/addon/ros_record.py

```diff
@@ -0,0 +1,33 @@
+from typing import List
+from threading import Lock
+
+from tycho_env.utils import GenericMessageSubscriber, print_and_cr, numpify
+
+
```

Comment:
rename this file? it is essentially relay messages from ros subscriber .. ros_record might confuse with logging functions